PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -949,7 +991,13 @@ void PMTraceConsumer::HandleWin32kEvent(EVENT_RECORD* pEventRecord)
         }
 
         PMTraceConsumer::Win32KPresentHistoryTokenKey key(CompositionSurfaceLuid, PresentCount, BindId);
+        // TODO: what if there is already an element in mWin32KPresentHistoryTokens?

```

Comment:
There won't be, since PresentCount is part of the key. If you add an assert, make sure that it doesn't impact retail code (i.e. lookups should be in the `assert()` expression or under `#ifdef`).